src/credentials/attestation_verifier/FileAttestationTrustStore.h

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <credentials/CHIPCert.h>
+#include <credentials/attestation_verifier/DeviceAttestationVerifier.h>
+
+#include <array>
+#include <vector>
+
+namespace chip {
+namespace Credentials {
+
+class FileAttestationTrustStore : public AttestationTrustStore
+{
+public:
+    FileAttestationTrustStore(const char * paaTrustStorePath);
+
+    CHIP_ERROR GetProductAttestationAuthorityCert(const ByteSpan & skid, MutableByteSpan & outPaaDerBuffer) const override;
```

Comment:
Suggest a `FileAttestationTrustStore`-specific method: `size_t GetNumProductAttestationAuthoritys() const`, which returns `mDerCertsRaw.size()`. This could be used for error-handling to detect if no certs are found.